index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+                <a href="#"class="my-logo">My Logo</a>
+                <nav class="nav-bar">
+                </nav>
+                <div class="menu-buttom"><a href="#"><img src="images/humburger-btn.svg" alt="menu-buttom"></a></div>
```

Comment:
- [ ] As the previous reviewer suggested, please wrap your hamburger menu with the `<nav>` tag, this gives your page a better semantic meaning since you're indicating that the hamburger is in charge of the navigation. 🤓 🔧 

```suggestion
                <nav class="nav-bar">
                  <div class="menu-buttom">
                    <a href="#">
                      <img src="images/humburger-btn.svg" alt="menu-buttom">
                    </a>
                  </div>
                </nav>
                
```